desktop/server/index.js

```diff
@@ -75,6 +75,21 @@ function startApp( started_cb ) {
 	} );
 }
 
+function isValidLastLocation( loc ) {
+	const invalids = [
+		'/desktop/hey.html',	// Page shown when no Electron
+		'/start'				// Don't attempt to resume the signup flow
+	];
+
+	for ( let s of invalids ) {
```

Comment:
A bit too tricky for my taste. ;-)
Nice and concise but hard to mentally parse what will get returned and why